CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/TeamsAPI/Models/TaskModuleInfo.cs

```diff
@@ -0,0 +1,99 @@
+// <auto-generated /> Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Bot.Connector.Teams.Models
+{
+    using System.Linq;
+
+    /// <summary>
+    /// Metadata for a Task Module.
+    /// </summary>
+    public partial class TaskModuleInfo
```

Comment:
Need to rename this to describe what this class is about. "Info" doesn't convey much.